lms/djangoapps/courseware/views.py

```diff
@@ -407,6 +408,19 @@ def _index_bulk_op(request, course_key, chapter, section, position):
             # Show empty courseware for a course with no units
             return render_to_response('courseware/courseware.html', context)
         elif chapter is None:
+            # Check to see if the course is gated on required content (such as an Entrance Exam)
+            # before access the course then redirect the view with entrance exam content e.g. chapter and section.
+            if settings.FEATURES.get('ENTRANCE_EXAMS', False) and getattr(course, 'entrance_exam_enabled', False):
+                exam_chapter, _ = get_entrance_exam_content_info(request, course_module, course)
```

Comment:
I think the convention here is to use double underscores as the single underscore is reserved for I18N, e.g. _('...')